src/coinstats.cpp

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2019 The Bitcoin Core developers
```

Comment:
In commit "move-only: move coins statistics utils out of RPC" (56ad8d226539a44daef02f479fab1820c536731f)

Maybe preserve the original copyright right years if you are moving existing code to a new file.